app/src/main/java/com/sopt/dive/component/navigation/BottomNavBar.kt

```diff
@@ -0,0 +1,72 @@
+package com.sopt.dive.component.navigation
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.size
+import androidx.compose.material3.NavigationBar
+import androidx.compose.material3.NavigationBarItem
+import androidx.compose.material3.NavigationBarItemDefaults
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.shadow
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.navigation.compose.currentBackStackEntryAsState
+import androidx.navigation.compose.rememberNavController
+import com.sopt.dive.navigation.TabItem
+import com.sopt.dive.ui.theme.DiveTheme
+
+@Composable
+fun BottomNavBar(navController: androidx.navigation.NavHostController) {
```

Comment:
```suggestion
fun BottomNavBar(navController: NavHostController) {
```
이렇게 작성해주구 import문 추가해주는게 더 깔끔띠 ~~~